### ISSUE TITLE
Dirac window is blank on startup in Dirac 1.2.38

### ISSUE THREAD

**Issue by aeberts** [binaryage/dirac#82]
Environment
Dirac Version 1.2.38
Chrome Canary Version : 71.0.3547.0
Mac: 10.11.6
Java: "1.8.0_101"

Issue: 
Dirac window is blank when starting Dirac 1.2.38 on above environment.

Expected behaviour: Dirac starts normally

Actual behaviour: When clicking Dirac extension button in Chrome the Dirac window appears but it is blank. The following errors are displayed when choosing: View >> Developer >> Developer Tools >> Console when the blank Dirac window is displayed:

```
shell.js:3130 [Deprecation] document.registerElement is deprecated and will be removed in M73, around March 2019. Please use window.customElements.define instead. See https://www.chromestatus.com/features/4642138092470272 for more details.
UI.registerCustomElement @ shell.js:3130
shell.js:3833 Uncaught RangeError: Maximum call stack size exceeded
    at Function.[Symbol.hasInstance] (<anonymous>)
    at $cljs$core$build_subvec$$ (shell.js:3833)
    at $cljs$core$Subvec$$.$JSCompiler_prototypeAlias$$.$cljs$core$IWithMeta$_with_meta$arity$2$ (shell.js:3833)
    at $cljs$core$_with_meta$$ (shell.js:3669)
    at $cljs$core$with_meta$$ (shell.js:3713)
    at $cljs$core$vec$$ (shell.js:3831)
    at $cljs$core$Subvec$$.$com$rpl$specter$impl$CoercePath$coerce_path$arity$1$ (shell.js:4439)
    at $com$rpl$specter$impl$coerce_path$$ (shell.js:4439)
    at $cljs$core$Subvec$$.$com$rpl$specter$impl$CoercePath$coerce_path$arity$1$ (shell.js:4439)
    at $com$rpl$specter$impl$coerce_path$$ (shell.js:4439)
devtools_app.js:2 Uncaught ReferenceError: SDK is not defined
    at devtools_app.js:2
inspector.js:2 Uncaught ReferenceError: SDK is not defined
    at inspector.js:2
```

Reproduction Steps:
* Update Dirac extension to 1.2.28
* Start Chrome Canary with remote debugging enabled
* Start re-frame application, nrepl and dirac agent as outlined in installation instructions
* Click Dirac extension icon in Chrome
* Result => Dirac page is displayed but the window is blank. The above mentioned error message is displayed in the Developer Tools console for the Dirac window.

Additional Notes:
* This project was working correctly with Dirac 1.2.27 on my laptop running Mac 10.13.6. 
* Restarted Chrome and did a `lein clean` and restarted figwheel servers => same result.

**Comment by darwin** [comment]
Thanks for reporting. I was able to reproduce the problem. It looks like something very weird happened when I compiled dirac in advanced mode. I was doing some experiments around issue #81 and maybe some cljs compiler caches stayed dirty when producing final build or something...

**Comment by darwin** [comment]
I just released a proper build as [v1.2.39](https://github.com/binaryage/dirac/releases/tag/v1.2.39).

It might take a while before updated version appears in chrome web store. In the meanwhile you might want to download the zip archive and drag&drop it onto your chrome extensions page.

---

Not sure what exactly happened, but I guess that changing cljs compiler from 1.10.339 to 1.10.415 (a local build) and then back to 1.10.339 and building advanced build caused it. Some compiler caches might have been incompatible/confused when produced by newer compiler and then consumed by old one. Just a theory. I did clean all caches I know about and new build seems to work as expected.
